### DISCUSSION TITLE
Variable-length LSTM in a mini-batch

### DISCUSSION THREAD

**Discussion by qqaatw** [apache/incubator-mxnet#14708] [Q&A]
Environment:
1. Python 3.6.3
2. Mxnet-cu92
3. Cuda 9.2 with cuDNN

I'm now proceeding a text processing networks, and have variable length inputs (time-steps) in a batch, I have already padded zeros if the sequence length is not enough to the max sequence length.

Is there a way to mask padded zeros and avoid it being updated?

p.s 
I saw [this](https://github.com/apache/incubator-mxnet/pull/3338) PR before, but I'm not sure that padded zeros would be updated or not.

[a comment by an automated account is not shown]

**Comment by frankfliu** [comment]
@mxnet-label-bot add [question, operator]
